python/pynessie/_log.py

```diff
@@ -27,26 +26,12 @@ def show_log(
     Note:
         limiting by path is not yet supported.
     """
-    raw_log = nessie.get_log(start)
+    raw_log = nessie.get_log(start_ref=start, author=author, committer=committer, before=before, after=after)
 
     def generator() -> Generator[CommitMeta, Any, None]:
         for i in raw_log:
-            if committer and i.committer != committer:
```

Comment:
Think, it's maybe better to keep this part and remove it in a future release. Not a big deal and I won't fight for not removing this.

Reply:
@rymurr wdyt? Should we keep this code for now?

Reply:
We should prob keep it w/ a note that it can be removed once the server has compatiblity guarantees. A new client on an old server wouldn't do any filtering if we remove it

Reply:
sgtm. fixed